src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanOrchestrator.hpp

```diff
@@ -257,47 +257,86 @@ class TScanOrchestrator final : public TOSPrimitives
+                    m_packageDeleteOrchestration->handleRequest(std::move(context));
+                    break;
+                case ScannerType::Os:
+                    logDebug2(
+                        WM_VULNSCAN_LOGTAG, "Processing 'OS scan' event for agent '%s'", context->agentId().data());
+                    m_osOrchestration->handleRequest(std::move(context));
+                    break;
                 case ScannerType::IntegrityClear:
+                    logDebug2(WM_VULNSCAN_LOGTAG,
+                              "Processing 'IntegrityClear' event for agent '%s'",
+                              context->agentId().data());
                     m_integrityClearOrchestration->handleRequest(std::move(context));
                     break;
                 // LCOV_EXCL_START
                 case ScannerType::ReScanAllAgents:
+                    logDebug2(WM_VULNSCAN_LOGTAG,
+                              "Re-scan all agents event received. Initiating re-scan for all agents");
                     m_reScanAllOrchestration->handleRequest(std::move(context));
                     m_eventDelayedDispatcher->clear();
                     break;
                 case ScannerType::ReScanSingleAgent:
+                    logDebug2(WM_VULNSCAN_LOGTAG,
+                              "Processing 'ReScanSingleAgent' event for agent '%s'",
+                              context->agentId().data());
                     m_reScanOrchestration->handleRequest(context);
                     m_eventDelayedDispatcher->clear(context->agentId());
                     break;
                 case ScannerType::CleanupAllAgentData:
+                    logDebug2(WM_VULNSCAN_LOGTAG, "Clean-up all data event received. Cleaning up data for all agents");
                     m_cleanUpDataOrchestration->handleRequest(std::move(context));
                     m_eventDelayedDispatcher->clear();
                     break;
                 case ScannerType::CleanupSingleAgentData:
+                    logDebug2(WM_VULNSCAN_LOGTAG,
+                              "Processing 'CleanupSingleAgentData' event for agent '%s'",
+                              context->agentId().data());
                     m_deleteAgentScanOrchestration->handleRequest(context);
                     m_eventDelayedDispatcher->clear(context->agentId());
                     break;
                 case ScannerType::GlobalSyncInventory:
+                    logDebug2(WM_VULNSCAN_LOGTAG,
+                              "Processing 'GlobalSyncInventory' event to synchronize inventory across nodes");
                     m_inventorySyncOrchestration->handleRequest(std::move(context));
                     break;
                 // LCOV_EXCL_STOP
                 default: return;
             }
-            logDebug2(WM_VULNSCAN_LOGTAG, "Event type: %d processed", type);
         }
+        logDebug2(WM_VULNSCAN_LOGTAG, "Event processed successfully");
```

Comment:
What events is processed successfully? We should return to the old log or put post "handleRequest" calls

```suggestion
        logDebug2(WM_VULNSCAN_LOGTAG, "Event type: %d processed", type);
```

Reply:
Done!